modules/users/client/utils/search.api.js

```diff
@@ -0,0 +1,5 @@
+import axios from 'axios';
+
+export async function searchUsers(query) {
```

Comment:
The current filename `search.api.js` doesn't match method name `searchUsers`:

How about we rename the file to `search-users.api.js`?

Reply:
yeah, I thought the same, but I also thought that maybe further methods might be added in the future and they might be living in the same source file, or am I wrong?

Reply:
Thanks for elaborating on the choice!

>further methods might be added in the future and they might be living in the same source file

If that happens, we can rename the file. Even with additional methods, it'll continue exposing "user search".

Generally speaking, it's better to not name files and methods "in case" but to accurately describe what they do right now. [YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it) and all that. 🙂

Reply:
Oh I suppose you were looking at the module name and thinking that no need to repeat that?

That would be valid point, but imagine if we'd have multiple "search" files open in the editor, it'll become pretty unwieldly fast.

Reply:
Done.